docs/src/inflation/terminology.md

```diff
@@ -31,7 +31,7 @@ The inflation rate actually observed on the Solana network after accounting for
 
 The rate of return (aka _interest_) earned on SOL staked on the network. It is often quoted as an annualized rate (e.g. "the network _staking yield_ is currently $10\%$ per year").
 
-- _Staking yield_ is of great interest to validators and token-holders holders who wish to delegate their tokens to avoid token dilution due to inflation (the extent of which is discussed below).
+- _Staking yield_ is of great interest to validators and token-holders who wish to delegate their tokens to avoid token dilution due to inflation (the extent of which is discussed below).
```

Comment:
Thanks! That hyphen looks out of place as well 
```suggestion
- _Staking yield_ is of great interest to validators and token holders who wish to delegate their tokens to avoid token dilution due to inflation (the extent of which is discussed below).
```